core/ustring.cpp

```diff
@@ -3071,6 +3071,7 @@ String String::repeat(int p_count) const {
 	const CharType *src = this->c_str();
 
 	new_string.resize(length() * p_count + 1);
+	new_string[length() * p_count] = 0;
```

Comment:
Should we make it `'\0'` to be more explicit?

Reply:
Not sure it's not clear enough this way, and all other cases of setting terminating zero are using ` = 0;` as well.

Reply:
Alright, consistency is the most important :)